variables.md

```diff
@@ -36,6 +36,7 @@ DUALBOOT | boolean | false | Enables dual boot configuration during the installa
 DUD | string | | Defines url or relative path to the DUD file if in [data directory of os-autoinst-distri-opensuse repo](https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/data)
 DUD_ADDONS | string | | Comma separated list of addons added using DUD.
 DVD |||
+ENABLE_DEPENDENCY_RESOLVER | boolean |       | Control when the resolve_dependecy_issues will be scheduled or not.
```

Comment:
default is false I guess

Reply:
i havent define somewhere default value. i mean there is no default value used anywhere. when the variable is used, it forces to do the scheduling of the resolve_dependency_issues or not. if itis not used in particular point it will schedule the module twice. so i dont think i need to set here anything.